spec/BowlingScorekeeperSpec.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+describe("BowlingScorekeeper", function() {
+  var bowlingScorekeeper;
+
+  beforeEach(function(){
+    bowlingScorekeeper = new BowlingScorekeeper();
+  });
+
+  it("allows a player to enter their name", function() {
+    bowlingScorekeeper.addName("Ollie")
+    expect(bowlingScorekeeper._playerName).toEqual("Ollie")
```

Comment:
This test should not be here - don't test the value of instance variables, that is part of the implementation. The test below is good instead, as you are testing a public method of the class.